_data/bitcoin.yml

```diff
@@ -60,6 +60,14 @@ websites:
       software: Yes
       doc: https://support.btc-e.com/index.php?/Knowledgebase/Article/View/97/25/how-do-i-set-2-factor-authentication-to-my-btc-e-account
 
+    - name: BTCJam
+      url: https://www.btcjam.com
+      img: btcjam.png
+      tfa: Yes
+      sms: Yes
```

Comment:
I don't see anything about sms more than that they verify your phone number upon initial TFA setup.